drivers/hmis/app/graphql/mutations/submit_assessment.rb

```diff
@@ -9,13 +9,15 @@ class SubmitAssessment < BaseMutation
     description 'Create/Submit assessment, and create/update related HUD records'
 
     argument :input, Types::HmisSchema::AssessmentInput, required: true
+    argument :assessment_lock_version, Integer, required: false
 
     field :assessment, Types::HmisSchema::Assessment, null: true
 
-    def resolve(input:)
+    def resolve(input:, assessment_lock_version: nil)
```

Comment:
How tricky do you think it would be to add this to `SubmitHouseholdAssessments`? I think it would need to accept an array of lock versions, one per assessment. I think we should probably do that, as those assessments should be locked too.

Reply:
It seems do-able, I'll give it a shot. I think we'd probably need to change it to an input array of [{assessment_id: , lock_version: }, ...]